eng/targetframeworksuffix.props

```diff
@@ -71,6 +71,7 @@
     </When>
     <When Condition="'$(TargetFrameworkSuffix)' == 'Browser'">
       <PropertyGroup>
+        <TargetsUnix>true</TargetsUnix>
         <TargetsLinux>true</TargetsLinux>
```

Comment:
Also delete `TargetsLinux` while you are on it? I do not think it really makes sense for WebAssembly to pretend to be Linux.

This was discussed in https://github.com/dotnet/corefx/pull/41777 and TODO comment got added, but it looks like the TODO was deleted without being fixed.

Reply:
Dine.